src/test/java/io/javalin/TestTrailingSlashes.kt

```diff
@@ -15,26 +15,26 @@ import org.junit.Test
 
 class TestTrailingSlashes {
 
-    private val nonIgnoringJavalin = Javalin.create().dontIgnoreTrailingSlashes()
-
-    @Test
-    fun `doesn't ignore when configured (instance)`() = TestUtil.test(nonIgnoringJavalin) { app, http ->
-        app.get("/hello") { ctx -> ctx.result("Hello, slash!") }
-        assertThat(http.getBody("/hello")).isEqualTo("Hello, slash!")
-        assertThat(http.getBody("/hello/")).isEqualTo("Not found")
-    }
-
-    @Test
-    fun `doesn't ignore when configured (ApiBuilder)`() = TestUtil.test(nonIgnoringJavalin) { app, http ->
-        app.routes {
-            path("a") {
-                get { ctx -> ctx.result("a") }
-                get("/") { ctx -> ctx.result("a-slash") }
-            }
-        }
-        assertThat(http.getBody("/a")).isEqualTo("a")
-        assertThat(http.getBody("/a/")).isEqualTo("a-slash")
-    }
+//    private val nonIgnoringJavalin = Javalin.create().servlet { ignoreTrailingSlahes = false }
```

Comment:
needs a fix or it's temporary commenting out?

Reply:
I'm not sure if we should remove the trailing slash config option. I haven't seen anyone use it though, so it's probably safe.